app/js/components/orderEntry/SearchAndAddOrder.jsx

```diff
@@ -26,6 +26,14 @@ export class SearchAndAddOrder extends React.Component {
           <Tab
             tabName="OutPatient">
             <SearchDrug />
+            {
+              this.props.drug.uuid ?
+                <AddForm
+                  drugName={this.props.drug.display}
+                  drugUuid={this.props.drug.uuid}
+                />
+                : <div />
```

Comment:
You can use the logical and operator, `&&` here, instead of the tenary. That way you wont have to hack inn an empty div